.gitignore

```diff
@@ -2,5 +2,6 @@
 .pnpm-store
 node_modules
 /.svelte
```

Comment:
you can remove this line now that `.svelte-kit` has been added

Reply:
83b8017